README.md

```diff
@@ -1 +1,73 @@
-# assignment-CoV2-analysis
+# Analysis Plan
+**By Andrea Chua**
+
+## Analysis 1: using kmer compositions to determine the changes in DNA composition relative to where the virus first began
+
+Virus first began in Wuhan, and data will be collected from NCBI.
+NCBI has the complete genome sequenced here [NCBI](https://www.ncbi.nlm.nih.gov/nuccore/1798174254).
+I am still in the process of finding more information,
+as NCBI doesn't seem to have all the information I need.
+I might need to tweak it to a specific geographical range/region
+to make things easier.
+
+
+### Functions Needed
+
+* Will be using Needleman-Wunsch Algorithm as the alignment algorithm
+but I am concerned with the amount of data that I am able to process with my code;
+but might also consider BioAlignments (below)
+
+* kmercount function from assignment04
+
+* calculating total kmers and then the percentage:
+
+    function kmerPercent(kmercount, kmerTotal)
+        total = (kmercount/kmerTotal) * 100 
+        return total
+    end
+
+* Perhaps refer to code in: [BioAlignments.pkg](https://github.com/BioJulia/BioAlignments.jl)
+
+    + look further into codes in [BioAlignments.jl package description](https://biojulia.net/BioAlignments.jl/stable/pairalign/)
+
+
+### Plots and Display Items
+
+* Will be using the Makie.jl to create graph
+
+* Radial plot of percentage of changes in kmer composition in relation to the distance it is from the origin 
+
+* A reference: ![radial plot](https://i.stack.imgur.com/sgiPd.png)
```

Comment:
This is very cool looking, but

1. I'm not totally clear on how the input data maps onto this plot and 
2. I'm not sure there's a ready-made recipe for this kind of plot. https://github.com/JuliaPlots/Makie.jl/issues/521